dll/win32/shell32/lang/pt-PT.rc

```diff
@@ -46,15 +46,15 @@ MENU_003 MENU
 BEGIN
     MENUITEM "&Retroceder\tCtrl+Z", FCIDM_SHVIEW_UNDO
     MENUITEM SEPARATOR
-    MENUITEM "Cor&tar\tCtrl+X", FCIDM_SHVIEW_CUT //FIXME: accelerator collision &T
+    MENUITEM "Cor&tar\tCtrl+X", FCIDM_SHVIEW_CUT
```

Comment:
I see no obvious drawback of what you suggest. Still the question: Is that same as 2k3sp2 pt-PT or just your gutfeeling? (Let's not guess on master head and play whack-a-mole in the future. Let's follow MS.)

Next question:
Is it correct to use &t here, but to use
`MENUITEM "C&ortar", IDM_CUT`
at line 67? It may be correct, I don't know for sure how pt-PT does that. Have no pt-PT Windows.

Line 49 deals with the edit-menu, and the line 67 with the context menu.

Make sure to check also IDS_CUT at the bottom of the file to match IDM_CUT. It doesn't match currently.

Reply:
Here is Win2003Srv